pkg/gator/fixtures/fixtures.go

```diff
@@ -500,13 +501,40 @@ apiVersion: admission.k8s.io/v1beta1
 request:
   operation: "DELETE"
   object:
+    kind: Pod
+    labels:
+      - app: "bar"
+`
+
+	DeleteAdmissionReviewWithOldObjectMissingKind = `
+kind: AdmissionReview
+apiVersion: admission.k8s.io/v1beta1
+request:
+  operation: "DELETE"
+  object:
```

Comment:
Do we want to emulate real admission requests for the DELETE operation such that 

 object is the new object being admitted.
  It is null for DELETE operations.

https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#webhook-request-and-response

Reply:
When I print `input.review.object` in the rego after running `kubectl delete` I definitely see the object and it is not null. Not sure if the doc is old. very strange.

Reply:
Hm, this is interesting. We specifically were interested in overwriting the `Object` when I/ we worked on this: https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/util/request_validation.go#L15-L32

Reply:
LOL looks like I added this in 2019 https://github.com/open-policy-agent/gatekeeper/pull/146 so it is a GK specific behavior. So for gator, we can assume `object` is not null then?

Reply:
IMO the input should match what we expect to receive from the API server (meaning an appropriate test would show object as nil and gator should handle the rewriting to make object == oldObject).

Reply:
>  an appropriate test would show object as nil and gator should handle the rewriting to make object == oldObject).

could I handle this in a followup PR?